packages/api-client/src/api/serializers/cart.ts

```diff
@@ -38,7 +38,7 @@ const deserializeLineItem = (lineItem: any, attachments: any[], config: any): Li
     : '';
 
   return {
-    _id: parseInt(lineItem.id, 10),
+    id: parseInt(lineItem.id, 10),
```

Comment:
@rafalcymerys using `parseInt` for IDs is not safe as it won't work with UUIDs which will be the default in the future :)